README.md

```diff
@@ -1,16 +1,16 @@
-# US Notify API
+# US Notify.gov API
```

Comment:
Isn't it now "Notify.gov" and not "US Notify.gov"

Reply:
Yes, thanks for flagging this!